CI/appveyor.install.sh

```diff
@@ -64,7 +64,8 @@ ROCKCOMMAND=${MINGW_INTERNAL_BASE_DIR}/bin/luarocks
     mingw-w64-${BUILDCOMPONENT}-hunspell \
     mingw-w64-${BUILDCOMPONENT}-zlib \
     mingw-w64-${BUILDCOMPONENT}-boost \
-    mingw-w64-${BUILDCOMPONENT}-yajl
+    mingw-w64-${BUILDCOMPONENT}-yajl \
+    mingw-w64-${BUILDCOMPONENT}-SDL2
```

Comment:
SDL2?

Reply:
Yeah, needed by Qt for the gamepad support I think.